docs/intro/tutorial01.txt

```diff
@@ -241,21 +256,9 @@ Your app directory should now look like:
         urls.py
         views.py
 
-In the ``polls/urls.py`` file include the following code:
-
-.. code-block:: python
-    :caption: ``polls/urls.py``
-
-    from django.urls import path
-
-    from . import views
-
-    urlpatterns = [
-        path("", views.index, name="index"),
-    ]
-
-The next step is to point the root URLconf at the ``polls.urls`` module. In
-``mysite/urls.py``, add an import for ``django.urls.include`` and insert an
+The next step is to configure the global URLconf in the ``mysite`` project to
```

Comment:
I'm not sure that changing "root URLconf" to "global URLconf" is an improvement. The setting is called `ROOT_URLCONF` and "root URLconf" is used elsewhere in the docs. I've never heard it called "global URLconf" before.

Reply:
Thank you! :star2: